src/election.js

```diff
@@ -138,35 +208,131 @@ export default class Election {
+
+        const stats = $(statsSelector);
+
+        if (!stats.length && config.debug) console.log(`missing stats elem (${statsSelector})`);
+
+        const text = stats.contents().map((_i, { data, type }) =>
             type === 'text' ? data.trim() : ""
         ).get().join(' ').trim();
+
+        const exprs = [
+            /(\d+(?:,\d+)+)\s+voters?\s+(?:were|was)\s+eligible/,
+            /(\d+(?:,\d+)+)\s+visited/,
+            /(\d+(?:,\d+)+)\s+visited(?:\s+the)\s+election/,
+            /(\d+(?:,\d+)+)\s+voted/
+        ];
+
+        const [eligible, visited, visitedElection, voted] = exprs.map(
+            (r) => +(r.exec(text) || [])[1].replace(/,/g, "")
+        );
+
+        return {
+            text,
+            eligible,
+            visited,
+            visitedElection,
+            voted
+        };
     }
 
     /**
-     * @param {import("./index.js").BotConfig} config
+     * @summary scrapes election winners
+     * @param {cheerio.Root} $ Cheerio root element
+     * @returns {number[]}
+     */
+    scrapeElectionWinnerIds($) {
+        const statsSelector = "#mainbar aside[role=status]:not(:nth-child(1)) .flex--item:nth-child(3) a";
+        return $(statsSelector).map(
+            (_i, el) => +($(el).attr('href').replace(/.+users\/(\d+)/, "$1"))
+        ).get();
+    }
+
+    /**
+     * @summary scrapes election site name
+     * @param {cheerio.Root} $ Cheerio root element
+     * @returns {string}
+     */
+    scrapeElectionSiteName($) {
+        return $('meta[property="og:site_name"]').attr('content').replace('Stack Exchange', '').trim();
+    }
+
+    /**
+     * @summary scrapes minimal rep to nominate in the election
+     * @param {cheerio.Root} $ Cheerio root element
+     * @returns {number}
+     */
+    scrapeElectionRepToNominate($) {
+        const conditionsNotice = $('#mainbar aside[role=status]').text();
+        const [, minRep = "0"] = /(?:more than )?(\d+,?\d+) reputation (?:may|to) nominate/m.exec(conditionsNotice) || [];
+        return +minRep.replace(/\D/g, "");
+    }
+
+    /**
+     * @summary scrapes minimal rep to voe in the election
+     * @param {cheerio.Root} $ Cheerio root element
+     * @returns {number}
+     */
+    scrapeElectionRepToVote($) {
+        const conditionsNotice = $('#mainbar aside[role=status]').text();
+        const [, minRep = "0"] = /(?:more than )?(\d+,?\d+) reputation (?:may|to) vote/m.exec(conditionsNotice) || [];
+        return +minRep.replace(/\D/g, "");
+    }
+
+    /**
+     * @summary scrapes election title
+     * @param {cheerio.Root} $ Cheerio root element
+     * @returns {string}
+     */
+    scrapeElectionTitle($) {
+        return $('#content h1').first().text().trim();
+    }
+
+    /**
+     * @summary scrapes election chat room URL
+     * @param {cheerio.Root} $ Cheerio root element
+     * @returns {string}
+     */
+    scrapeChatURL($) {
+        const chatRoomLinkSelector = "#mainbar .s-prose a[href*=\"/rooms/\"]";
```

Comment:
Just a personal preference, but if a string is going to have double quotes in them I'd prefer to use single or backticks instead of escaping:

```
'#mainbar .s-prose a[href*="/rooms/"]'
`#mainbar .s-prose a[href*="/rooms/"]`
```

Reply:
Oh, no problem with that - it's just I have the IDE set up this way - can adjust with an editorconfig so as we also do not override each other's settings unnecessarily.